e2e-tests/demand-backup/run

```diff
@@ -118,7 +118,42 @@ fi
 
 if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
 	check_backup_deletion "https://s3.amazonaws.com/operator-testing/${backup_dest_aws}" "aws-s3"
-	#	check_backup_deletion "https://storage.googleapis.com/operator-testing/${backup_dest_gcp}" "gcp-cs"
+	check_backup_deletion "https://storage.googleapis.com/operator-testing/${backup_dest_gcp}" "gcp-cs"
+	check_backup_deletion "https://engk8soperators.blob.core.windows.net/operator-testing/${backup_dest_azure}" "azure-blob"
+fi
+
+desc 'checking backup deletion without cr'
+run_backup minio
+if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
+	run_backup aws-s3
+	run_backup gcp-cs
```

Comment:
Why we can do it step by step:
```
run_backup minio
wait_backup "$backup_name_minio"
if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
	run_backup aws-s3
	run_backup gcp-cs
	run_backup azure-blob
        wait_backup "$backup_name_aws"
	wait_backup "$backup_name_gcp"
	wait_backup "$backup_name_azure"
fi

```

Reply:
We can, but this way then we run_backup minio and wait_backup minio immediately and then run backups on other storages and wait for them.
If we want first to run backups on all storages and then wait for all of them, then we can't do it like this.